app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  after_create :update_counter
+  after_destroy :update_counter
+
+  def update_counter
+    post.comments_count = post.comments.count
+    post.save
+  end
```

Comment:
- [ ] **[OPTIONAL]** As a suggestion, I think this custom method should be private.